python/revng/cli/revng.py

```diff
@@ -10,10 +10,11 @@
 
 
 def extend_list(list, new_items):
-    new_items_set = set(new_items)
+    # new_items_set = set(new_items)
```

Comment:
I see you're commenting code here and there.
We do not commit commented code. If it's dead code, drop it.

Reply:
Dropped